1-weekly-mission-week2-main/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Linkbrary</title>
+    <meta property="og:title" content="Linkbrary" key="title" />
+    <meta
+      name="description"
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+      key="description"
+    />
+    <meta
+      property="og:image"
+      content="https://visitbusan.net/uploadImgs/files/cntnts/20211130150754165_wufrotr"
+      key="image"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@linkbrary" />
+    <meta name="twitter:title" content="Linkbrary" />
+    <meta
+      name="twitter:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+    />
+    <meta
+      name="twitter:image"
+      content="https://visitbusan.net/uploadImgs/files/cntnts/20211130150754165_wufrotr"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/landing.css?after" />
+  </head>
+  <body>
+    <nav>
+      <div class="gnb">
+        <a href="index.html">
+          <img
+            class="linkbrary-logo"
+            src="./images/logo.svg"
+            alt="홈으로 연결된 Linkbrary 로고"
+          />
+        </a>
+        <a class="cta cta-short" href="signin.html">
+          <span>로그인</span>
+        </a>
+      </div>
+    </nav>
+    <header>
+      <div class="hero-header">
```

Comment:
@JaeBeen95 

[질문]
개인 취향이지만 개인적으로는 hero-header 라는 이름은 `<hedaer>` 태그에 있어야 하지 않을까 싶어요.

```suggestion
    <header id="hero-header">
      <div class="hero-header-div">
```

항상 네이밍이 어렵죠. 음.. div 태그 네이밍을 한다면... 다음과 같이 생각이 드네요.
- hero-header-div
- hero-header-section
- hero-header-wrapper
- hero-header__div
- hero-header__box
- hero-header-container
- hero-header__container